src/preact/context.js

```diff
@@ -59,3 +59,11 @@ export function WithAmpContext(props) {
 
   return <AmpContext.Provider children={props['children']} value={current} />;
 }
+
+/**
+ * @return {JsonObject}
+ */
+export function useAmpContext() {
```

Comment:
Rebase. This should no longer be needed.

Reply:
Thanks updated!